gallery/src/styles/NxReadOnly/NxReadOnlyPage.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+const nxReadOnlyDlExampleCode = require('!!raw-loader!./NxReadOnlyDlExample.html').default;
+
+const NxReadOnlyPage = () => {
+  return (
+    <>
+      <GalleryDescriptionTile>
+        <p className="nx-p">
+          <code className="nx-code">.nx-read-only</code> is a design pattern for when you wish to display data in a
+          read only form. It's basic structure is a label and one or more data points.
+        </p>
+        <p className="nx-p">
+          <code className="nx-code">.nx-read-only</code> is layed out using a description list
+          (<code className="nx-code">&lt;dl&gt;</code>) as the container.
+        </p>
+        <table className="nx-table nx-table--gallery-props">
```

Comment:
I've been trying to use `NxTable` on new pages.  At the very least, `.nx-table--gallery-props` no longer does anything as far as I'm aware.

Reply:
> I've been trying to use `NxTable` on new pages. At the very least, `.nx-table--gallery-props` no longer does anything as far as I'm aware.

Ah right: https://github.com/sonatype/sonatype-react-shared-components/pull/243/commits/9ad4504c99bb00bebc4b42ac5b3a468f96f8b540